terminus-ui/select/src/select.component.scss

```diff
@@ -25,6 +25,12 @@ $ts-select-placeholder-arrow-space: 2 * ($ts-select-arrow-size + $ts-select-arro
   @include reset;
   display: block;
   outline: none;
+  .ts-form-field {
+    .ts-form-field__infix {
+      $select-form-field-padding: .6em;
```

Comment:
We should probably either a) add `-top` to to this variable name, or b) move all the padding numbers up into the definition.

Reply:
I was actually wondering whether you'd call this out lol

Also, your overall comment made me think some about `input` component stuff is required when first glanced at it!